infra/app.py

```diff
@@ -8,17 +8,37 @@
 
 from aws_cdk import core
 from datalake.datalake_stack import DataLakeStack  # pylint: disable=import-error
+from datalake.networking_stack import NetworkingStack  # pylint: disable=import-error
```

Comment:
How do we avoid having to annotate all these lines? Do we need to move some files?

Reply:
No idea. This is layout created by CDK.